main.py

```diff
@@ -12,19 +12,19 @@
 from disvae.encoder import EncoderBetaB
 from disvae.decoder import DecoderBetaB
 from disvae.training import Trainer
-from utils.dataloaders import (get_mnist_dataloaders, get_dsprites_dataloader,
+from utils.dataloaders import (get_dataloaders, get_mnist_dataloaders, get_dsprites_dataloader,
                                get_chairs_dataloader, get_fashion_mnist_dataloaders,
                                get_img_size)
 from viz.visualize import Visualizer
 
 
-def default_config():
-    return {'epochs': 10,
+def default_experiment():
+    return {'epochs': 100000,
             'batch_size': 64,
             'no_cuda': False,
             'seed': 1234,
             'log-level': "info",
-            "lr": 1e-3,
+            "lr": 5e-4,
```

Comment:
is that what they use (and do they use adam) ? the normal value for adam is 1e3. But maybe for VAE people use smaller :)

Reply:
Yeah Adam is usually closer to 1e-3 but they report 5e-4

Reply:
I made this change in my latest commit